src/common/low_precision_transformations/src/concat.cpp

```diff
@@ -55,17 +55,22 @@ bool ConcatTransformation::transform(TransformationContext& context, ngraph::pat
     }
 
     bool allDequantizationShiftAreZero = true;
+    bool allDequantizationShiftConvertAreNotZero = true;
     bool allDequantizationMultiplyAreZero = true;
     for (const auto& dequantization : layerDequantizations) {
         if (dequantization.subtract != nullptr) {
             allDequantizationShiftAreZero = false;
+            if (dequantization.subtractConvert == nullptr) {
+                allDequantizationShiftConvertAreNotZero = false;
+            }
         }
 
         if (dequantization.multiply != nullptr) {
             allDequantizationMultiplyAreZero = false;
         }
 
-        if (!allDequantizationShiftAreZero && !allDequantizationMultiplyAreZero) {
+        if (!allDequantizationShiftAreZero && !allDequantizationMultiplyAreZero &&
+            !allDequantizationShiftConvertAreNotZero) {
             break;
         }
     }
```

Comment:
`allDequantizationShiftConvertAreNotZero` variable initialization is not correct. If there are no `Substract` operations  then `allDequantizationShiftConvertAreNotZero` will be `true`.

Reply:
Now if allDequantizationShiftAreZero is true after loop then set allDequantizationShiftConvertAreNotZero false. Thank you